scripts/examples/03-Machine-Learning/00-TensorFlow/tf_object_detection.py

```diff
@@ -50,7 +51,7 @@ def fomo_post_process(model, inputs, outputs):
     n, oh, ow, oc = model.output_shape[0]
     nms = NMS(ow, oh, inputs[0].roi)
     for i in range(oc):
-        img = image.Image(outputs[0], shape=(oh, ow, 1), strides=(i, oc), scale=(0, 1))
+        img = image.Image(outputs[0][0, :, :, i] * 255)
```

Comment:
Notice how we are building an image out of a sliced ndarray here and scaling it. Magic.

Reply:
That is indeed magic :)